analyses/tp53_nf1_score/README.md

```diff
@@ -1,6 +1,6 @@
 ## Apply classifiers trained on TCGA RNA-seq data
 
-**Module author:** Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6))
+**Module author:** edited Greg Way @gwaygenomics' code by Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6)) Chante Bethell (@cbethell) and Jaclyn Taroni (@jaclyn-taroni)
```

Comment:
```suggestion
**Module author:** Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6)) and Jaclyn Taroni (@jaclyn-taroni); code adapted from Gregory Way ([@gwaygenomics](https://github.com/gwaygenomics))
```
Using https://github.com/AlexsLemonade/OpenPBTA-analysis/commits/master/analyses/tp53_nf1_score as a starting point and then looking at the commits that include scientific decision making.